extension/lib/doc.js

```diff
@@ -103,6 +107,13 @@ class Doc {
                 let end = start + sentenceBoundary.offset;
                 this.sentences.push(new SentencePointer(container_id, start, end));
             }
+            if (this.sentences.length > MAX_NUM_SENTENCE) {
+                debug("Too many sentences found = " + this.sentences.length);
```

Comment:
Should we possibly alert this to them, so they're not confused if this happens?

Reply:
Yeah good point. Hmm.. for now the quick options are either:
1. alert popup
2. console log

I say alert cos altho unsightly, at least they won't think that the app is busted.
Also, not that annoying cos we tell them just once per big page and they'll learn.